### DISCUSSION TITLE
Hello! Is there a standard method to subset the bam file from cellranger with seurat_clusters?

### DISCUSSION THREAD

**Discussion by methornton** [satijalab/seurat#8452] [Q&A] [answered]
Hello!

I was wondering if there was a "standard" method for doing this?  I found this blog that was very helpful and I think it would be a really good idea and feature for looking at the frequency of your UMI and then even reads for low expressed DE UMI, with the UCSC genome browser.  So I wanted to subset by the SCT clusters and then the integrated clusters from the raw bam file. If you have any ideas or comments they are greatly appreciated.

Here is the blog URL, https://divingintogeneticsandgenomics.com/post/split-a-10xscatac-bam-file-by-cluster/

Thanks!

**Comment by samuel-marsh** [comment, marked as answer]
Couple of tools (I would also note that just from very Quick Look I think that blog post should also be applicable to GEX BAMs too).

https://kb.10xgenomics.com/hc/en-us/articles/360022448251-How-to-filter-the-BAM-file-produced-by-10x-pipelines-with-a-list-of-barcodes

https://github.com/10XGenomics/subset-bam/issues/17#issuecomment-1917084971

https://github.com/10XGenomics/subset-bam

https://github.com/timoast/sinto

Best,
Sam
